src/core/runtime.d

```diff
@@ -328,45 +390,78 @@ struct Runtime
      *
      *             if (fp)
      *             {
+     *                 ++result.executed;
      *                 try
      *                 {
      *                     fp();
+     *                     ++result.passed;
      *                 }
      *                 catch (Throwable e)
      *                 {
      *                     writeln(e);
-     *                     failed++;
      *                 }
      *             }
      *         }
      *     }
-     *     return failed == 0;
+     *     if(result.executed != result.passed)
```

Comment:
space after if, mmmmkay? :)

Reply:
You could use torture on me, and I still won't ever remember to do this 😆